client/my-sites/plugins/plugin-details-CTA/preinstalled-premium-plugins-CTA.jsx

```diff
@@ -65,6 +72,18 @@ export default function PluginDetailsCTAPreinstalledPremiumPlugins( {
 		</>
 	);
 
+	const includedWithAtomicPlanButton = (
+		<div className="plugin-details-cta__install">
+			<Button
+				className="plugin-details-cta__install-button"
+				href={ `/checkout/${ selectedSiteSlug }/${ preinstalledPremiumPluginProduct }` }
```

Comment:
I think we can replace `href` with an `#` or an empty string if this button is always disabled

Reply:
Yes agreed. Good point! 👍